src/collections/Events.ts

```diff
@@ -44,17 +44,44 @@ export const Events: CollectionConfig = {
           displayFormat: "dd.MM.yyyy",
         },
       },
+      validate: (value: string, options) => {
+        if (options.siblingData.start > value) {
+          return "Das Enddatum muss nach dem Startdatum sein.";
```

Comment:
What about single day events?

Reply:
Equal start and end dates are allowed because it gives error when start > end not >=.
But thats not reflected in the error message, so I'll update it.

Reply:
Maybe change the order of the check then? You're right, I read it wrong so maybe it would be clearer if it's

```suggestion
if (value < options.siblingData.start) {
          return "Das Enddatum kann nicht vor dem Startdatum sein.";
```

But date comparisons always feel unituitive to me so `¯\_(ツ)_/¯`